frontend/src/pages/ConnectedPages/Game/UsersInGame/index.tsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import { UserInGame } from "../../../../websockets/MessageType";
+
+import "./usersingame.css";
+
+interface IUsersInGame {
+  users: UserInGame[],
+}
+
+const maxUsersToDisplay = 14;
+const maxCharactersPerName = 8;
+
+const UsersInGame: React.FC<IUsersInGame> = ({ users: users }: IUsersInGame): JSX.Element => {
+  const totalPlayers = users.length;
+  const totalColumns = 3;
+
+  const setRowClassName = (index: number): string => {
+    return (index + 1) % 2 ? "usersInGameEvenRowColor" : "usersInGameParRowColor";
+  }
```

Comment:
We can remove this by adding css to the body like this:

```
.trClass:nth-of-type(2n) {
    background-color: #00000014;
}

.trClass:nth-of-type(2n+1) {
    background-color: #00000030;
}
```
This or the other way for the colors, I am not sure. You can check the description of `:nth-of-type` [here](https://developer.mozilla.org/fr/docs/Web/CSS/:nth-of-type).
A close relative would be `:nth-child`

We can change this too in the `Leaderboard/index.tsx` file